### ISSUE TITLE
UnicodeDecodeError 

### ISSUE THREAD

**Issue by wouter89** [facebook/infer#354]
Command & Console output::
```
infer -- mvn package

Traceback (most recent call last):
  File "/Users/wstikkelorum/Applications/homebrew/bin/infer", line 178, in <module>
    main()
  File "/Users/wstikkelorum/Applications/homebrew/bin/infer", line 142, in main
    capture_exitcode = imported_module.gen_instance(args, cmd).capture()
  File "/Users/wstikkelorum/Applications/homebrew/Cellar/infer/0.8.0/libexec/infer/lib/python/inferlib/capture/mvn.py", line 80, in capture
    cmds = self.get_infer_commands(util.get_build_output(self.build_cmd))
  File "/Users/wstikkelorum/Applications/homebrew/Cellar/infer/0.8.0/libexec/infer/lib/python/inferlib/capture/util.py", line 27, in get_build_output
    return verbose_out_chars.split('\n')
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 242946: ordinal not in range(128)
```

toplevel.log
```
[INFO] Running command /Users/wstikkelorum/Applications/homebrew/bin/infer -- mvn package
[INFO] Path to infer script /Users/wstikkelorum/Applications/homebrew/bin/infer (/Users/wstikkelorum/Applications/homebrew/Cellar/infer/0.8.0/libexec/infer/lib/python/infer)
[INFO] Infer version v0.8.0
Copyright 2009 - present Facebook. All Rights Reserved.


[INFO] Platform: Darwin-15.4.0-x86_64-i386-64bit
[INFO] PATH=/Users/wstikkelorum/Applications/infer/examples/infer/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/usr/local/munki:/usr/local/scripts/lab:/usr/local/scripts/monitor:/usr/local/scripts/lab/customer/kasbank:/usr/local/scripts/uploads:/lab/Sat-LATEST/Networks/bin:/lab/dbcompare:/lab/Crawler-LATEST/bin:/lab/CloudTools-LATEST:/Library/TeX/texbin:/Users/wstikkelorum/Applications/homebrew/bin:/Users/wstikkelorum/Applications/infer/infer/bin
[INFO] SHELL=/bin/bash
[INFO] PWD=/Users/wstikkelorum/Documents/workspace/MIaaS
[INFO] Apache Maven 3.2.3 (33f8c3e1027c3ddde99d3cdebad2656a31e8fdf4; 2014-08-11T22:58:10+02:00)
Maven home: /usr/local/libexec
Java version: 1.8.0_91, vendor: Oracle Corporation
Java home: /Library/Java/JavaVirtualMachines/jdk1.8.0_91.jdk/Contents/Home/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.11.4", arch: "x86_64", family: "mac"
```

locale:
```
LANG="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_CTYPE="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_ALL="en_US.UTF-8"
```

Tried solutions from #320 : 
1. Running infer with: LC_ALL="en_US.UTF-8" infer ...
2. Changing "/usr/local/Cellar/infer/0.8.0/libexec/infer/lib/python/inferlib/capture/gradle.py" 
(3. no non-ascii characters in pwd)

But, none are fixing the problem. 
Is there something else I can look at to fix this problem?

**Comment by jvillard** [comment]
This should be fixed in d4521fd70d9cb9545943f5ce2d84cd410fe8b0e8. Can you please try the latest master and re-open this issue if there is still a problem? Thanks for the report!

**Comment by wouter89** [comment]
I updated to version 8.1 but the problem still occurs. (Same as I reported above) 

Is there anything I can look for in the codebase to solve the problem or create a work-around?

**Comment by jvillard** [comment]
The fix was published after 0.8.1, and we haven't made a new release yet. In the meantime, you can [compile the latest master from source](https://github.com/facebook/infer/blob/master/INSTALL.md#install-infer-from-source).
